src/web/javascript/loadingPage.js

```diff
@@ -0,0 +1,107 @@
+const code = document.getElementById("code");
+
+let time = 200;
+
+code.classList.remove("borderBlink");
+setTimeout(() => code.innerHTML += "<span style='color: black;'>c</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>o</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>n</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>s</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>o</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>l</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>e</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>.</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>l</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>o</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>g</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>(</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "'", time);
+time += 170;
+setTimeout(() => code.innerHTML += "l", time);
+time += 170;
+setTimeout(() => code.innerHTML += "o", time);
+time += 170;
+setTimeout(() => code.innerHTML += "a", time);
+time += 170;
+setTimeout(() => code.innerHTML += "d", time);
+time += 170;
+setTimeout(() => code.innerHTML += "i", time);
+time += 170;
+setTimeout(() => code.innerHTML += "n", time);
+time += 170;
+setTimeout(() => code.innerHTML += "g", time);
+time += 170;
+setTimeout(() => code.innerHTML += "'", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>)</span>", time);
+time += 170;
+setTimeout(() => code.innerHTML += "<span style='color: black;'>;</span>", time);
```

Comment:
Is building statements like this the only way?
What if we want to make changes in the future? Would we have to manipulate these many lines, letter by letter?
A better way would have it in a way that accepts a string in a single variable, or a handful of variables and applies the whole loading logic automatically.

Reply:
I am aware of this, and I have thought of a way. But for merge this, will change it later because it is not a priority.

Reply:
for now*

Reply:
other issues can wait, please fix this.